ushahidi/platform-client
# Load scripts when needed

This pull request makes the following changes:
- Removes Facebook-sdk (was not used)
- Loads Intercom-script when logging in instead of for all users
- Removes old doorbell script

Testing checklist:
- Go to share-> select Facebook
- [x] You should get directed to Facebook (i.e no changes to current behaviour, it should still work)
- Make sure you are logged out and localstorage is cleared
- Make sure you have an app-id in config.js
- Login
- [x] Intercom-script should load (check network tab)]

- [x] Intercom bubble should appear (if new messages to show)

- Logout
- [x] Chatbox should disappear
- Login again
- [x] Intercom-script is already present in window and should not load again.
- [x] Intercom chatbox should appear
- [x] Intercom bubble should appear (if new messages to show)

Fixes ushahidi/platform# .

Ping @ushahidi/platform
